appveyor.yml

```diff
@@ -6,14 +6,15 @@ environment:
 install:
   - SET PATH=%PYTHON%;%PYTHON%\\Scripts;%JAVA_HOME%\bin;%PATH%
   - pip install --upgrade setuptools
-  - pip install py2exe_py2
+  - pip install http://sourceforge.net/projects/py2exe/files/latest/download?source=files
```

Comment:
Why the full URL? Explain with a comment.

Reply:
Says in the top of the PR description: 
> Change py2exe package origin back to sourceforge in hopes of fixing random HTTPS package error

When I download the released sysl.exe I get the same error that Blair gets `'module' object has no attribute 'HTTPSConnection'` however with a freshly built one on my system I do not get that error. The only difference other than potential os & architecture differences is where py2exe came from....

Reply:
It used to be this. Anyway, I'll add the comment.
`py2exe` doesn't exit on PyPI for Python 2.7....

Reply:
Done